apps/testing-javascript/src/components/pricing-section.tsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react'
 import Image from 'next/image'
 import cx from 'classnames'
+import {max} from 'lodash'
 import type {CommerceProps} from '@skillrecordings/commerce-server/dist/@types'
 import {useCoupon} from '@skillrecordings/skill-lesson/path-to-purchase/use-coupon'
 import {PriceCheckProvider} from '@skillrecordings/skill-lesson/path-to-purchase/pricing-check-context'
 import Balancer from 'react-wrap-balancer'
 
 import {Pricing} from 'path-to-purchase/pricing'
 
+const PRODUCTS_RANKS = {
+  'kcd_da6ab36c-b091-4f6f-90aa-d7db2fc798ff': 1,
+  'kcd_fb976b99-0633-4329-bbfb-f5f76dc278b3': 2,
+  'kcd_4f0b26ee-d61d-4245-a204-26f5774355a5': 3,
+}
+
```

Comment:
Hardcoded tiers and related ranks

Reply:
Though there is probably some clever way to use the `UpgradableProducts` table to derive this ranking, I think this is probably a better and quicker approach to the same result 💯